resources/badcert.html

```diff
@@ -3,6 +3,24 @@
 <title>Certificate error</title>
 </head>
 <body>
-    <p>${reason}</p>
+  <p>${reason}</p>
+  <pre id="bytes">${bytes}</pre>
+  <button id="allow">Allow certificate temporarily</button>
+  <script>
+    let bytes = document.getElementById('bytes').textContent;
+    let button = document.getElementById('allow');
+    if (bytes.length) {
+        button.onclick = function() {
+            let xhr = new XMLHttpRequest();
+            xhr.open('GET', 'chrome:allowcert?secret=${secret}&bytes=' + btoa(bytes));
```

Comment:
I'm a bit concerned that we're using a GET request for this, side-effecting GET that adds a security wormhole scares me!

Reply:
What is the security wormhole, exactly?

Reply:
Yeah IMO this should be a POST. Secrets in url params is bad